e2e-tests/cypress/integration/appeal-submisssion-email-notification.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Send email notification
```

Comment:
Is it not a duplicate with appeal-submission-email-confirmation.feature ?

Reply:
Lots of duplication ready for refactoring on another day.